drivers/hmis/app/graphql/types/hmis_schema/assessment.rb

```diff
@@ -11,6 +11,10 @@ class HmisSchema::Assessment < Types::BaseObject
     include Types::HmisSchema::HasDisabilityGroups
     include Types::HmisSchema::HasCustomDataElements
 
+    available_filter_options do
+      arg :roles, [Types::Forms::Enums::FormRole]
```

Comment:
This is to spec, just noting that it doesn't actually show the correct list. We only want to show "assessment" roles.

Would you mind creating a new enum called `AssessmentType`, and changing this to

```suggestion
      arg :assessment_types, [Types::Forms::Enums::AssessmentType]
```

It should be pretty straightforward:
```
// on definition model
use_enum_with_same_key :assessment_type_enum_map, HUD_ASSESSMENT_FORM_ROLES

// new class

module Types
  class Forms::Enums::AssessmentType < Types::BaseEnum
    graphql_name 'AssessmentType'

    with_enum_map Hmis::Form::Definition.assessment_type_enum_map
  end
end
```

<img width="381" alt="Screen Shot 2023-05-23 at 9 15 10 AM" src="https://github.com/greenriver/hmis-warehouse/assets/5333982/7edcce9f-1d40-46e3-a2e0-32fd93c024aa">

In the medium-term this will likely be changed to `arg :assessment_types, [ID]` so that we can include custom assessment types in the filter. But that isn't planned yet, so let's get this looking appropriate first with an  enum just for hud assessments.